src/store/redusers/get_data_reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { SET_DATA } from '../constants';
+
+const initialState = {
+  inverter_one: [],
+  inverter_two: [],
+};
+
+export const set_data_reducer = (state = initialState, action: { type: string; value: any; }) => {
```

Comment:
from get_data_reducer you are exporting set_data_reducer. That's a bit weird. Please make it consistent

Reply:
done